server/.env

```diff
@@ -1,2 +1,2 @@
-TEAM_NAME="Shums,Tony Fang"
```

Comment:
we can remove the `TEAM_NAME` variable entirely it's not necessary. Also make sure you gitignore your `.env`